src/walletExplorer.ts

```diff
@@ -0,0 +1,254 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as vscode from 'vscode';
+
+import { IWallet } from './iWallet';
+import { wallet } from '@cityofzion/neon-core';
+
+const NewWalletFileInstructions = 'Save this JSON file anywhere in your workspace. You will then see your new wallet appear ' +
```

Comment:
Is there a standard way to handle localization in VSCode extensions we should be using?

Reply:
I opened https://github.com/neo-project/neo-visual-tracker/issues/4 to track this (a long time ago) but haven't done any investigation yet.  I can look into that soon, it will be more wide reaching than this PR though (I have English UI strings in a lot of other existing pieces of code).